src/components/Keyboard/SwitchKeyboard/SwitchKeyboard.css

```diff
@@ -0,0 +1,80 @@
+.keyboardContainer {
+    text-align: center;
+}
+
+.changeKeyboardButton {
+    color: #191919;
+    cursor: pointer;
+    height: 40px;
+    width: 250px;
+    font-size: 14px;
+    white-space: nowrap;
+
+    @media (min-width: 576px) {
+        height: 40px;
+        width: 350px;
+        font-size: 20px;
+    }
+}
+
+.qwertyContainer {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    padding: 12px;
+
+    & .qwertyPart1, & .qwertyPart2, & .qwertyPart3, & .latinLetters {
+        & .letter {
+            font-size: 20px;
+        }
+        padding: 5px 0px;
+        grid-auto-rows: 35px;
+        grid-gap: 12px;
+        grid-template-columns: repeat(10, 35px);
+    }
+    & .qwertyPart2 {
+        grid-template-columns: repeat(9, 35px);
+        padding-right: 15px;
+    }
+    & .qwertyPart3 {
+        grid-template-columns: repeat(7, 35px);
+        padding-right: 60px;
+        padding-bottom: 15px;
+    }
+    $ .latinLetters {
+        grid-template-columns: repeat(9, 35px);
+
+    }
+    @media (min-width: 576px) {
+        & .qwertyPart1, & .qwertyPart2, & .qwertyPart3, & .latinLetters {
+            & .letter {
+                font-size: 25px;
+            }
+            grid-auto-rows: 43px;
+            grid-template-columns: repeat(10, 43px);
```

Comment:
To avoid repetition(of css in general) you can use [css variables](https://www.w3schools.com/css/css3_variables.asp)

I tested option where you can replace hard coded value e. g. `grid-template-columns: repeat(7, `**`35px`**`);` with `grid-template-columns: repeat(7, `**`var(--query-layout-width`**`));` and then you could change only value of that variable inside of the media query e. g. 
```css
@media(min-width: 576px) {
    --query-layout-width: 43px;
}
```
You define css variable in special `:root` pseudoselector:
```css
:root {
    --query-layout-width: 35px;
}
```

Reply:
Thanks for good idea :+1: